src/components/storage-plan-modal.js

```diff
@@ -70,6 +70,7 @@ function StoragePlanModal({ isOpen, storagePlan, storagePlanNamesInUse, onOk, on
       onCancel={onCancel}
       maskClosable={false}
       okButtonProps={{ loading }}
+      forceRender
```

Comment:
This causes a hydration error (only when the page starts on the storage-plans` tab)